tests/chainer_tests/test_variable.py

```diff
@@ -2329,6 +2329,53 @@ def test_backward_chainerx(self):
         self.check_backward(chainerx.array(self.x))
 
 
+@testing.parameterize(*testing.product({
+    'shape': [(), (1,), (1, 1), (1, 1, 1), (2,), (2, 3)],
```

Comment:
Why don't you test empty shapes (`x.size == 0`)?

Reply:
I missed them. Thank you.